en/installation-guide/workstation.rst

```diff
@@ -93,6 +93,8 @@ You can install them on debian based systems with::
  sudo pip install networkx
  sudo apt-get install libqt4-sql-psql
 
+How to install networkx on non debian based systems (Windows, macOS?)?
```

Comment:
Please do not add questions into the documentation. There is some documentation about that here: https://github.com/QGEP/QGEP/discussions/689

Reply:
Thanks Stefan. I found pip3 install networkx in the meantime.
I thought so, that adding questions into the documentation is not a good idea. How should I ask one?

Reply:
@decheu I have transfered your issue to the discussion: https://github.com/QGEP/QGEP/discussions/714